packages/paste-website/src/pages/components/timeline/index.mdx

```diff
@@ -0,0 +1,334 @@
+Use [Skeleton Loader](/components/skeleton-loader) for results that might take a while to load.
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutText>
+    Tip: For very long Timelines, where pressing 'Load more' Buttons might become cumbersome, consider using a{" "}
+    <Anchor href="/components/side-modal">Side Modal</Anchor> with infinite scroll to display the complete list of
+    events.
+  </CalloutText>
+</Callout>
+
+<ResponsiveImage src={InfiniteTimeline} alt="a demo page with filter pattern" />
+
+
+## Composition notes
+
+A Timeline should be presented in chronological order. The order can either be descending (most recent event at the top the list) or ascending (oldest event at the top of the list)
+
+**Use descending order on a Timeline when the current event is the most relevant information for the user.**
+
+<UnorderedList>
+  <ListItem>
+    Time sensitive information, for example an <strong>SMS sending tracker</strong> where the final status (sent or failed) is
+    crucial for the user to track in real time.
+  </ListItem>
+  <ListItem>
+    Long lists of events that are constantly updated, for example in <strong>customer call logs</strong> where the most recent
+    interaction is displayed first so agents can access the information in a timely manner.
+  </ListItem>
+</UnorderedList>
+
+**Use ascending order on a Timeline when understanding the progression from the oldest event to the most recent is essential.**
+
+<UnorderedList>
+  <ListItem>
+    Diagnosing an issue, for example in <strong>platform issue reporting</strong>, where understanding the sequence of reported
+    issues is necessary to understand the problem.
+  </ListItem>
+  <ListItem>
+    Events that evolve through time, for example in <strong>payment history</strong> where the price has changed over time.
+  </ListItem>
+</UnorderedList>
+
+Any content and other components can be placed within the Timeline item detail.
+
+<LivePreview scope={{ Timeline, TimelineItem, StatusBadge, Box, Paragraph, Anchor, Table, THead, TBody, Td, Th, Tr }} noInline>
+  {TimelineComposition}
+</LivePreview>
+
+### Timestamp
+
+Follow Paste's [date guidelines](/foundations/content/product-style-guide#dates) and [time guidelines](/foundations/content/product-style-guide#time) to format the timestamp for each Timeline item.
+In use cases where more than one timestamp per Timeline item is required, use separators in between timestamps for clarity. Order multiple timestamps by starting with the largest time unit and ending with the smallest:
+
+<OrderedList>
+  <ListItem>Month and Day, Year (ex: "August 10, 2024")</ListItem>
+  <ListItem>Hour, Minute, Second (ex: "12:38 pm")</ListItem>
+  <ListItem>Additional Details (ex: "0.2 s" for duration)</ListItem>
+</OrderedList>
+
+<LivePreview scope={{ Timeline, TimelineItem, StatusBadge, Box, Paragraph }} noInline>
+  {TimelineTimestamp}
+</LivePreview>
+
+## When to use Timeline
+
```

Comment:
**issue (blocking):**  It seems something happened with the examples, the "Event titles" labels are placed on the description, and the 3rd one has only one timelineItem.
![Screenshot 2024-09-17 at 3 59 16 PM](https://github.com/user-attachments/assets/368bbd28-601f-4153-bd21-7cbad33e99b6)